src/util/run-as-root.js

```diff
@@ -28,10 +28,13 @@ const sudoPromptToRunShell = async cmd =>
         sudoPrompt.exec(cmd, opts, (error, stdout, stderr) => {
             debug(`sudo ${cmd} returned`, { error, stdout, stderr });
             if (error) {
-                error.message = [error.message, stderr, stdout]
-                    .filter(x => !!x)
-                    .join('\n\n');
-                return reject(error);
+                return reject(
+                    new Error(
+                        [error.message || error, stderr, stdout]
+                            .filter(x => !!x)
```

Comment:
Nitpick: can't this be simplified to `.filter(x => x)`?

Reply:
Can also be `.filter(Boolean)`